pkg/node/volumemgr.go

```diff
@@ -67,6 +67,8 @@ const (
 
 	fakeAttachAnnotation = "pv.attach.kubernetes.io/ignore-if-inaccessible"
 	fakeAttachAllowKey   = "yes"
+
+	driveHealthOverrideAnnotation = "health"
```

Comment:
please put comments for this annotations

Reply:
added